DataFormats/TauReco/interface/RecoTauPiZero.h

```diff
@@ -14,32 +14,57 @@ class RecoTauPiZero : public CompositePtrCandidate {
       kStrips = 3
     };
 
-    RecoTauPiZero():CompositePtrCandidate(),algoName_(kUndefined){
-      this->setPdgId(111); }
-
-    RecoTauPiZero(PiZeroAlgorithm algoName):
-        CompositePtrCandidate(), algoName_(algoName) { this->setPdgId(111); }
+    RecoTauPiZero()
+      : CompositePtrCandidate(),
+        algoName_(kUndefined)
```

Comment:
initialize data members here instead of the body of the constructor (for all simple initializations)